xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -88,6 +93,8 @@ def _should_skip_target(*, ctx, target):
     """
     if ctx.rule.kind in _BUILD_TEST_RULES:
         return True
+    if ctx.rule.kind in _TEST_SUITE_RULES:
+        return True
```

Comment:
I think this, and the replacement labels stuff, can be two separate PRs.

- Replacement labels: can add an example to `examples/integrations` for an `ios_unit_test` with multiple runners (or just adjust one of existing ones to have multiple runners).
- `test_suite`: Can add a `test_suite` around two tests in `examples/integration` and list it in `top_level_targets`

Reply:
Ok, after we reverted the approach of bringing all runner variations into Xcode I thought this could go back to being a single change but I could be missing something:

>Replacement labels: can add an example to examples/integrations for an ios_unit_test with multiple runners (or just adjust one of existing ones to have multiple runners).

I think `ios_unit_test` [only takes a single runner](https://github.com/bazelbuild/rules_apple/blob/master/doc/rules-ios.md#ios_unit_test)? 

>test_suite: Can add a test_suite around two tests in examples/integration and list it in top_level_targets

That would be adding `ios_unit_test_suite` and `ios_ui_test_suite` with `runners` set and then `rules_ios`'s `ios_unit_test` with `runners` set (`rules_ios` call the above two under the hood). IIUC this already covers what you intended to cover in the previous item making it unnecessary?

Reply:
Got confused. Can land `test_suites` support first (just raw `test_suite`, not `ios_unit_test_suite`), then this change with the `replacement_labels` fix can come after to add support for `ios_unit_test_suite`/`rules_ios`'s `ios_unit_test.runners`.

Reply:
Ok thx for clarifying, I had only the `ios_*` rules in mind, didn't realize we could (and should) test `test_suite` rules explicitly first. Decoupling things like this makes sense from the perspective of `rules_xcodeproj` regardless how each repo abstracts these, thx for clarifying.

Will split this into the two PRs then!

Reply:
+1 to supporting test_suite. On Cash we use the native rule along with multiple ios_unit_test and `target_compatible_with` to split up our snapshot tests by device. 

Each unit test target specifies a runner and they're all part of the same suite. Ideally, this would show up as a single target in Xcode where the runner is whatever Xcode has selected. 

Right now I see all 4 targets get loaded and running it spins up the entire test suite, which spins up 4 sims + 1 sim from Xcode

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2184

Just one Q there about custom schemes